packages/unmock-core/src/backend/index.ts

```diff
@@ -101,7 +104,14 @@ export default class NodeBackend {
   }
 
   public get services(): ServiceStoreType {
-    return this.serviceStore;
+    return (this.serviceStore || NodeBackend.dummyStore).services;
```

Comment:
I don't really understand the use for dummy store here, why not return `this.serviceStore && this.serviceStore.services || {}`?

Reply:
Just a typing shortcut, since typescript complains that an empty object does not contain the `services` key.